agent/connect.sh

```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
+RUNNER_APPLICATION_DIRECTORY=$HOME/agent/
+source ./utils.sh
+source $RUNNER_APPLICATION_DIRECTORY/.env
 
-user=$(curl -s "http://169.254.169.254/metadata/github_user" | \
-        python3 -c "import sys, json; print(json.load(sys.stdin)['value'])")
-repo=$(curl -s "http://169.254.169.254/metadata/github_repo_name" | \
-        python3 -c "import sys, json; print(json.load(sys.stdin)['value'])")
-vm_name=$(curl -s "http://169.254.169.254/metadata/orka_vm_name" | \
-        python3 -c "import sys, json; print(json.load(sys.stdin)['value'])")
-pat=$(curl -s "http://169.254.169.254/metadata/github_pat" | \
-        python3 -c "import sys, json; print(json.load(sys.stdin)['value'])")
+user=`fetch_encrypted_metadata github_user`
```

Comment:
I suppose nobody else is using this, but after this change, it will only support encrypted metadata and we have a new dependency on jq.

I was thinking that the method of fetching metadata should be provided in the image and used by this, with a default fallback that fetches metadata unencrypted without a jq dependency.

Reply:
Removed the method `fetch_encrypted_metadata `